01/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import DICTIONARY, LETTER_SCORES
+
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY) as file:
+        words = file.read().strip().split('\n')
```

Comment:
do not use split with `\n` instead use `splitlines`. \n is not correct for all OS